src/utils/searchUtils.ts

```diff
@@ -191,8 +205,8 @@ export function mapQueryToSearchParams({
     contributor,
     title,
     subject,
-    sortBy: sort,
-    order: sort_direction,
+    sortBy: sort as SortKey,
```

Comment:
Why not set `sortBy` as `SortKey` in the `SearchQueryParams` interface so you don't have to do the `... as ...` type coercion? Same for `sort_direction`.